specs/capri/Multi-attach.md

```diff
@@ -133,4 +133,55 @@ Drivers will need to add a capabilities field "multiAttach: True/False" to their
 
 ## References
 
-https://github.com/openstack/cinder-specs/blob/master/specs/queens/enable-multiattach.rst#id2
\ No newline at end of file
+https://github.com/openstack/cinder-specs/blob/master/specs/queens/enable-multiattach.rst#id2
+
+## Driver configuration yaml file example for Multi-Attach
+
+```
+authOptions:
+  username: "admin"
+  password: "IaaS@PORTAL-CLOUD9!"
+  # Whether to encrypt the password. If enabled, the value of the password must be ciphertext.
+  EnableEncrypted: false
+  # Encryption and decryption tool. Default value is aes. The decryption tool can only decrypt the corresponding ciphertext.
+  PwdEncrypter: "aes"
+  url: "https://8.46.195.74:28443"
```

Comment:
Is this IP address real?  If so, please change to a fake one.

Reply:
sure